integrationTests/cypress/support/commands/usermgt_commands.js

```diff
@@ -0,0 +1,284 @@
+// ***********************************************
+// commands for the usermgt
+// ***********************************************
+
+// ***********************************************
+// REST - /api/account
+// ***********************************************
+
+/**
+ * Return the user account that is authenticated by the given credentials.
+ * @param {String} username - The username of the account.
+ * @param {String} password - The password of the account.
+ */
+const usermgtGetAccount = (username, password) => {
+    cy.request({
+        method: "GET",
+        url: Cypress.config().baseUrl + "/usermgt/api/account",
+        auth: {
+            'user': username,
+            'pass': password
+        }
+    }).then((response) => {
+        expect(response.status).to.eq(200)
+        return response.body
+    })
+}
+
+// ***********************************************
+// REST - /api/users
+// ***********************************************
+
+/**
+ * Return the user defined by the username.
+ * @param {String} username - The username of the user.
+ */
+const usermgtGetUser = (username) => {
+    cy.fixture("ces_admin_data.json").then(function (admindata) {
+        cy.request({
+            method: "GET",
+            url: Cypress.config().baseUrl + "/usermgt/api/users/" + username,
+            auth: {
+                'user': admindata.username,
+                'pass': admindata.password
+            }
+        }).then((response) => {
+            expect(response.status).to.eq(200)
+            return response.body
+        })
+    })
+}
+
+/**
+ * Creates the user defined in in the given user data object parameter.
+ * A failed request is not tolerated and fails the test.
+ * @param {String} username - The username for the user.
+ * @param {String} givenname - The real given name of the user.
+ * @param {String} surname - The real surname of the user.
+ * @param {String} displayName - The displayname for the user.
+ * @param {String} mail - The E-Mail for the user.
+ * @param {String} password - The password for the user.
+ */
+const usermgtCreateUser = (username, givenname, surname, displayName, mail, password) => {
```

Comment:
displayName is camelcase, the rest is lowercase

Reply:
Given that "username" has become a closed-form compound word in common English usage, following the camelCase guidelines' recommendation would mean camel casing the word as "username" rather than "userName".

The same applies to surname, and also password :)